### PR TITLE
Fix tests #2

### PR DESCRIPTION
**Pull request by chx2**

### CODE REVIEW

File: src/Policies/KeywordTypePolicy.php [hunk 1]
```diff
@@ -1,49 +0,0 @@
-<?php
```
Comment (chx2) [inline src/Policies/KeywordTypePolicy.php:1]: Removed this policy, I don't believe these are included in the migration anymore either

**Comment by drewroberts** [comment]
Great branch name, PR title, and commit message. Thanks!

File: src/Policies/KeywordTypePolicy.php (hunk 1 above)
Comment (drewroberts) [reply src/Policies/KeywordTypePolicy.php:1]: That's correct. Thanks

**Review by drewroberts — APPROVED** [review]
Thank you! ✳️

**Pull request merged**